Applications/_Libs/CMP_GUI/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+
+add_library(CMP_GUI INTERFACE)
```

Comment:
This target represents libraries that are in `Applications/CompressonatorGUI` that are used outside of that tool. Those files should be moved into a common space if they are depended on by other things.